src/DXRMapFixups/DeusEx/Classes/DXRFixupVandenberg.uc

```diff
@@ -1105,6 +1110,11 @@ function AnyEntryMapFixes()
             //Dump the current state to see if a TriggerToggle door got changed to TriggerOpenTimed or something
             l(dxm$": " $ dxm.GetStateName() @ dxm.bStasis @ dxm.KeyNum @ dxm.bUseTriggered @ dxm.bDamageTriggered @ dxm.SavedTrigger);
         }
+
+        if (dxr.flagBase.GetBool('schematic_downloaded')) {
+            player().StartDataLinkTransmission("DL_Everett_Congrats");
```

Comment:
Is there any reason this would play when re-entering the main part of the lab, and not as part of the conversation Savage already has with you right after you get the schematics?

Does it actually add anything of value, or is it just more people talking at you over the infolink?

As a point of reference, even Revision doesn't appear to have restored this infolink.

Reply:
You already merged those two Jock conversations